.goutputstream-JXD5EZ

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+import MySQLdb
+#import peewee
+#from peewee import *
+db = MySQLdb.connect(host="localhost", user="SHIFT", passwd="%&12SHilaE7", db="autopen")
```

Comment:
I don't think that including the username and password of our server is a good idea here. This would give anyone access to the database.